packages/server/lib/util/replace_stream.ts

```diff
@@ -0,0 +1,79 @@
+const GraphemeSplitter = require('grapheme-splitter')
+const through = require('through')
+
+const splitter = new GraphemeSplitter()
+
+/**
+ * UTF-8 grapheme aware stream replacer
+ * https://github.com/cypress-io/cypress/pull/4984
+ */
+function replaceStream (patterns: RegExp | RegExp[], replacements: string | string[], options = { maxTailLength: 100 }) {
+  if (!Array.isArray(patterns)) {
+    patterns = [patterns]
```

Comment:
maybe this is typescript foo but can't you just do `patterns = [].concat(patterns)` without the check on the `if(...)` ?

Reply:
If you do that, TS complains because it expects the first arg to `.concat` to be an array, but patterns could be a RegExp.